### PR TITLE
Add `teamNames` collection.

### PR DESCRIPTION
**Pull request by dmccraw**
## Add `teamNames` collection

**Story / Task Link**
<https://teamsnap.atlassian.net/browse/MNG-3939>

#### Details
- Add the `teamNames` collection
- Bump version to 1.52.0

### CODE REVIEW

**Comment by amberwebb** [comment]
Other than changing the date in the changelog, looks good to me. 

@matthopson What do you think?

**Comment by dmccraw** [comment]
@amberwebb updated

File: docs/collections/team_names.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<a id="loadTeamNames"></a>
+## `loadTeamNames(params, callback)`
+Loads items from the `TeamNames` collection based on given params.
+
+### Params
+* `params`: [int, object] - can be either a `teamId` or an object with query parameters.
+* `callback`: [function] - callback to be executed when the operation completes.
+
+To see a list of all available search params you can run:
+`teamsnap.collections.TeamNames.queries.search.params`
```
Comment (matthopson) [inline docs/collections/team_names.md]: Should be `teamsnap.collections.teamNames` (case)

File: docs/collections/team_names.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# TeamNames
```
Comment (matthopson) [inline docs/collections/team_names.md]: We usually space this here `Team Names`

File: docs/collections/team_names.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Methods
+
+- [loadTeamNames](#loadTeamNames)
+
+
+---
+<a id="loadTeamNames"></a>
+## `loadTeamNames(params, callback)`
+Loads items from the `TeamNames` collection based on given params.
```
Comment (matthopson) [inline docs/collections/team_names.md]: case

**Comment by dmccraw** [comment]
@matthopson updated. Will merge and deploy in the morning

**Pull request merged**